test/integration/workload/test.sh

```diff
@@ -30,51 +30,64 @@ workload_test() { # WORKLOAD_KIND
     echo "::endgroup::"
     single_test "w_${KIND}_${APIVERSION}_${TAG}" "Testing ${KIND} using ${APIVERSION} and ${TAG} image..." "workload" "${KIND}" "default" " created" "null"
 
-    if [[ "${GITHUB_BASE_REF:-"no-github-base-ref-set"}" == "master" || "${GITHUB_BASE_REF:-"no-github-base-ref-set"}" == "no-github-base-ref-set" ]]; then
+    if [[ "${GITHUB_BASE_REF:-"no-github-base-ref-set"}" == "master" || "${GITHUB_BASE_REF:-"no-github-base-ref-set"}" == "no-github-base-ref-set" || true ]]; then
```

Comment:
was || true from testing? bc this is always true

Reply:
yes just for testing